client/src/templates/Challenges/components/challenge-transcript.tsx

```diff
@@ -14,11 +15,28 @@ function ChallengeTranscript({
 }: ChallengeTranscriptProps): JSX.Element {
   const { t } = useTranslation();
 
+  // default to expanded
+  const [isOpen, setIsOpen] = useState(
+    () => (store.get('fcc-transcript-expanded') as boolean | null) ?? true
```

Comment:
What's the consensus for this?

Reply:
Seems like most people don't want them open by default.

Reply:
so, doesn't this need to be changed to false?

```suggestion
    () => (store.get('fcc-transcript-expanded') as boolean | null) ?? false
```

Reply:
yes, or something like that. Most of the people that weighed in did like them collapsed by default, but Quincy said he wanted them open by default. So I'm not sure what to do - I was going to ask more of the team and see what they think and maybe run it by Quincy again if that's the consensus. So for now, we can just block this.